app/api/testing/firebase-test-utils.ts

```
@@ -0,0 +1,147 @@
+import { FirebaseApp } from 'firebase/app';
+import { User } from 'firebase/auth';
+import { Firestore } from 'firebase/firestore';
+import { FirebaseApiClient } from '../FirebaseApiClient';
+import { ApiInterface, UserInfo } from '../ApiInterface';
+
+/**
+ * Mock FirebaseApiClient for testing
+ * Implements the same interface but allows mocking of all methods
+ */
+export class MockFirebaseApiClient implements ApiInterface {
+  private connected: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)